Makefile

```diff
@@ -33,12 +33,16 @@ verify:
 
 .PHONY: build
 build:
-	$(GO_CMD) build -o bin/kwok ./cmd/controller/*.go
+	$(GO_CMD) build -o bin/kwok ./cmd/kwok/*.go
 
 .PHONY: integration-test
 integration-test:
 	@echo "Not implemented yet"
 
 .PHONY: e2e-test
-e2e-test:
-	@echo "Not implemented yet"
+e2e-test: \
+	e2e-test-kwok
+
+.PHONY: e2e-test-kwok
+e2e-test-kwokctl:
```

Comment:
can you make this part of `e2e-test` as there is no CI job for `e2e-test-kwok`.

Reply:
Ooops, e2e didn't work before

``` console
[1](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A1)
./test/kwok/kwok.test.sh
[2](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A2)
./test/kwok/../../images/kwok/build.sh: line 18: docker: command not found
[3](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A3)
./test/kwok/kwok.test.sh: line 24: kind: command not found
[4](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A4)
./test/kwok/kwok.test.sh: line 26: kind: command not found
[5](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A5)
./test/kwok/kwok.test.sh: line 28: kubectl: command not found
[6](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A6)
./test/kwok/kwok.test.sh: line 28: kubectl: command not found
[7](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kwok/11/pull-kwok-e2e-test-main/1557193367748612096#1:build-log.txt%3A7)
./test/kwok/kwok.test.sh: line 33: kubectl: command not found
```

docker, kind, kubectl , none of them, I don't know what to do.

Reply:
This case deploying `kwok` in a `kind` cluster and checking if it works.
I ignore the e2e-test to change the name first.

Reply:
> Maybe we need to change the base image of the ci

Yes, the current image is a golang based one. Please submit a PR in test-infra to use separate image.